### ISSUE TITLE
Improve speed

### ISSUE THREAD

**Issue by Ch4nKyy** [AlexofTheBigE/Lego-Mindstorm-EV3-Basispraktikum-WS-18-19-Gruppe-5#4]
- Line following
- Transition from line following to box push
- Color search

The difficulty is that changing speeds often breaks the logic. Be careful.

**Comment by Mavros-Lykos** [comment]
yeah. if you use more sensors and speed microcontroller, it would be accurate even at high speed. but it also have a limit !
